src/backend/distributed/executor/multi_router_executor.c

```diff
@@ -1340,7 +1359,8 @@ ExtractParametersFromParamListInfo(ParamListInfo paramListInfo, Oid **parameterT
  */
 static bool
 StoreQueryResult(CitusScanState *scanState, MultiConnection *connection,
-				 bool failOnError, int64 *rows)
+				 bool failOnError, int64 *rows,
+				 DistributedExecutionStats *executionStats)
```

Comment:
We could consider putting rows in executionStats as well.

Reply:
I can do it but should we do it in this PR as a separate commit or does it deserve its own PR?